lib/Parser/RegexCompileTime.cpp

```diff
@@ -1281,6 +1281,7 @@ namespace UnifiedRegex
         {
             Char uniqueEquivs[CaseInsensitive::EquivClassSize];
             CharCount uniqueEquivCount = FindUniqueEquivs(cs, uniqueEquivs);
+            Assert(uniqueEquivCount >= 2);
             switch (uniqueEquivCount)
             {
             case 2:
```

Comment:
Maybe defensively add a case 1? (`EMIT(compiler, MatchChar2Inst, uniqueEquivs[0])`

Reply:
Unfortunately, there's no MatchChar1Inst; I've made it a failfast, however.